mtags/src/main/scala/scala/meta/internal/metals/ExactCharSequence.scala

```diff
@@ -15,11 +15,11 @@ case class ExactCharSequence(value: CharSequence) extends CharSequence {
       // this charsequence when adding it to a bloom filter.
       12345.toChar
     } else {
-      value.charAt(index + 1)
+      value.charAt(index - 1)
     }
   }
 
   override def subSequence(start: Int, end: Int): CharSequence = {
-    ExactCharSequence(value.subSequence(start + 1, end + 1))
+    ???
```

Comment:
Should this be left at just `???` ?

Reply:
Yes, this method should not be called.

Reply:
I can replace this with a `throw new UnsupportedOperationException`